test/OpenTelemetry.Instrumentation.AspNetCore.Tests/BasicTests.cs

```diff
@@ -1211,6 +1211,55 @@ void ConfigureTestServices(IServiceCollection services)
 
         Assert.Empty(hubActivity);
     }
+
```

Comment:
What do you think about?

```suggestion
#endif

#if NET10_0_OR_GREATER
```

Reply:
In the other PR I suggested the current state so what we have in main today is still compiling this code (and testing it does nothing) to ensure it doesn't get accidentally broken before we actually use the .NET SDK.

I can add .NET 10 specific positive tests to my PR that adds net10.0.